mmomni/ansible/playbooks/jitsi/reconfigure.yml

```diff
@@ -0,0 +1,291 @@
+            url: https://download.jitsi.org/jitsi-key.gpg.key
+            state: present
+        - name: "Add prosody repo key"
+          apt_key:
+            id: 107D65A0A148C237FDF00AB47393D7E674D9DBB5
+            url: https://prosody.im/files/prosody-debian-packages.key
+            state: present
+        - name: "Add jitsi repository"
+          apt_repository:
+            repo: deb https://download.jitsi.org stable/
+            state: present
+        - name: "Add prosody repository"
+          apt_repository:
+            repo: deb https://packages.prosody.im/debian focal main
+            state: present
+        - name: "Configure /etc/hosts"
+          lineinfile:
+            path: /etc/hosts
+            regexp: '^127\.0\.0\.1 omnibus-jitsi'
+            line: 127.0.0.1 omnibus-jitsi {{ jitsi_fqdn }} {{ jitsi_fqdn.split(".") | first }}
+            state: present
+            owner: root
+            group: root
+            mode: '0644'
+        - name: "Install jitsi"
+          apt:
+            pkg:
+              - jitsi-meet-web
+              - jitsi-meet-prosody
+              - jicofo
+              - coturn
+              - dnsutils
+              - jitsi-videobridge2
+            state: present
+        - name: "Create jitsi config directory"
+          file:
+            path: /etc/jitsi/meet
+            state: directory
+            mode: 0755
+        - name: "Configure jitsi meet"
+          template:
+            src: jitsi-config.js
+            dest: /etc/jitsi/meet/config.js
+            owner: root
+            group: root
+            mode: 0644
+        - name: "Configure jitsi meet interface"
+          template:
+            src: jitsi-interface-config.js
+            dest: /etc/jitsi/meet/interface_config.js
+            owner: root
+            group: root
+            mode: 0644
+        - name: "Configure Jitsi videobridge"
+          template:
+            src: videobridge.config
+            dest: /etc/jitsi/videobridge/config
+            owner: root
+            group: root
+            mode: 0644
+        - name: "Configure the jitsi video bridge sip communicator"
+          template:
+            src: videobridge.sip-communicator
+            dest: /etc/jitsi/videobridge/sip-communicator.properties
+            owner: root
+            group: root
+            mode: 0644
+        - name: "Configure Jitsi jicofo"
+          template:
+            src: jicofo.config
+            dest: /etc/jitsi/jicofo/config
+            owner: root
+            group: root
+            mode: 0644
+        - name: "Configure prosody"
+          template:
+            src: prosody.cfg.lua
+            dest: "/etc/prosody/conf.d/{{jitsi_fqdn}}.cfg.lua"
+            owner: root
+            group: root
+            mode: 0644
+        - name: "Register prosody jvb user"
+          shell: "prosodyctl unregister jvb auth.{{ jitsi_fqdn }}; prosodyctl register jvb auth.{{ jitsi_fqdn }} {{ jitsi_jvb_secret }} || true"
```

Comment:
I think you can skip the unregister, registering will overwrite the existing credentials.